### ISSUE TITLE
Reviews blocks: author and date text doesn't have enough contrast

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#1676]
Text color is `#808080` and background is white, so there [isn't enough contrast](https://webaim.org/resources/contrastchecker/?fcolor=808080&bcolor=FFFFFF).

**Screenshots**
![imatge](https://user-images.githubusercontent.com/3616980/73386987-17e90780-42d0-11ea-8575-230a0a7cae07.png)

**Comment by nerrad** [comment]
Possible this might be a theme issue?

**Comment by Aljullu** [comment]
It looks like the color is hardcoded on our side:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/assets/js/base/components/reviews/review-list-item/style.scss#L142

This issue might be a good opportunity to completely remove the `color` declaration from that selector. If we want the block to look ok in all theme backgrounds, it should use the text color inherited from the theme.
